systemd/asus-fan-control.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Fan control for ASUS devices running Linux
```

Comment:
Is using the dot at the end of this line an issue? If so, we can (afterwards, not now) create an issue and make everything dot-less... :smile:

Reply:
I don't think it is, so pushed the change